modules/gitops-argo-cd-dynamic-scaling.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argo-cd-dynamic-scaling_{context}"]
+= Enabling dynamic scaling of shards of the Argo CD Application Controller
+
+By default, the Argo CD Application Controller assigns clusters to shards indefinitely. If you are using the `round-robin` sharding algorithm, this static assignment can result in uneven distribution of shards, particularly when replicas are added or removed. You can enable dynamic scaling of shards to automatically adjust the number of shards based on the number of clusters managed by the Argo CD Application Controller at a given time. This ensures that shards are well balanced and optmizes the use of compute resources.
+
+[IMPORTANT]
+====
+Dynamic scaling of shards is a Technology Preview feature only. Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete. Red{nbsp}Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope].
+====
+
+.Prerequisites
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {OCP} web console.
+* You have installed the {gitops-title} Operator in your cluster.
```

Comment:
Move the prerequisites under the new created sections described in the following comments. Thank you!

Reply:
ACK.

Addresed!

Thank you.